TYPO3.Flow/Classes/TYPO3/Flow/Mvc/Controller/AbstractController.php

```diff
@@ -196,8 +196,6 @@ protected function forward($actionName, $controllerName = null, $packageKey = nu
         }
         if ($packageKey !== null) {
             $nextRequest->setControllerPackageKey($packageKey);
-        }
```

Comment:
Right, subpackageKey is not a method argument, but is directly derived from packageKey.

Test needs to be adjusted though, and there seems to be an unrelated CGL error - should maybe be rebased

Also, does this maybe make sense for 2.3/3.0 too?